### ISSUE TITLE
accessibility focus issue

### ISSUE THREAD

**Issue by zslabs** [metafizzy/flickity#296]
Hi all,
Currently if you attempt to interact with the prev/next buttons while the slider is not in a favorable viewport position, the page jumps you down to the slider, but doesn't actually plumb-through the action you clicked on. For example, if I click the next arrow while it is not "in-view", then I am jumped to the slider, but it does not pass through the next click event as I expect, so I have to click the next button again.

I'm having trouble finding where this is done in the source, but don't want to have to turn off `accessibility` (which resolves the issue) because that also negates keyboard nav. Thanks!

**Comment by desandro** [comment]
It's happening in [`Flickity.prototype.pointerDownFocus` in drag.js](https://github.com/metafizzy/flickity/blob/v1.1.1/js/drag.js#L150).

After looking deeper into this, I've added a hack to reset the scroll position after the element has been focused. The next version of Flickity (after v1.1.1) will have this fix.

You can shim in the hack by adding this code:

``` js
var touchStartEvents = {
  touchstart: true,
  MSPointerDown: true
};

var focusNodes = {
  INPUT: true,
  SELECT: true
};

Flickity.prototype.pointerDownFocus = function( event ) {
  // focus element, if not touch, and its not an input or select
  if ( !this.options.accessibility || touchStartEvents[ event.type ] ||
      focusNodes[ event.target.nodeName ] ) {
    return;
  }
  var prevScrollY = window.pageYOffset;
  this.element.focus();
  // hack to fix scroll jump after focus, #76
  if ( window.pageYOffset != prevScrollY ) {
    window.scrollTo( window.pageXOffset, prevScrollY );
  }
};
```

**Comment by zslabs** [comment]
Thanks @desandro works great, appreciate you looking into this and will take out this hack once the new version of Flickity is out.
